### DISCUSSION TITLE
Using MongoDB Atlas Search

### DISCUSSION THREAD

**Discussion by imunisasi** [jenssegers/laravel-mongodb#2636] [Q&A] [answered]
Hello , is this possible to search using this one :
https://www.mongodb.com/atlas/search

any example for this , or just using normal search ?

thanks

**Comment by GromNaN** [comment, marked as answer]
Yes, it is possible using a raw aggregation. There is an example in the mongodb library: https://github.com/mongodb/mongo-php-library/blob/0017da3d475a7795cb0577f4991b385992e51573/examples/atlas-search.php#L98

To use with laravel:

```php
use MongoDB\Laravel\Collection;

YourModel::raw(function (Collection $collection) {
    return $collection->aggregate([
        [
            '$search' => [
                'index' => 'default',
                'text' => [
                    'query' => <text>,
                    'path' => [<field>],
                ],
            ],
        ],
        ['$limit' => 10],
    ]
}));
```

**Reply by imunisasi** [reply]
thanks a lot @GromNaN , 
in Atlas search there is a score in result
how to showing score in this result ?

**Reply by GromNaN** [reply]
I don't know Atlas Search enough to answer. You can look at the [`$search` documentation](https://www.mongodb.com/docs/atlas/atlas-search/query-syntax/). For Laravel integration, you can add an attribute to your model to receive this the score value (don't save it).
